<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOutPage</class>
 <widget class="QWidget" name="CheckOutPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="PERSON_1">
       <property name="title">
        <string>入住人</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="LE_NAME_1">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>手机号：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MobilePhoneLineEdit" name="LE_PHONE_1">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>身份证号：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="IDNumberLineEdit" name="LE_ID_1">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="TIME">
       <property name="title">
        <string>住店时间</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>入住时间:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DateTimeEdit" name="CB_CHECKIN_DATE"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>离店时间:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="DateTimeEdit" name="CB_CHECKOUT_DATE"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="LE_ROOM">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>房间号：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox">
       <property name="title">
        <string>实际费用</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="LE_FEE">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="PB_FIND">
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_CHECKOUT">
         <property name="text">
          <string>退房</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QComboBox</extends>
   <header>Widgets/DateTimeEdit/date_time_edit.h</header>
  </customwidget>
  <customwidget>
   <class>MobilePhoneLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/MobilePhoneLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>IDNumberLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/IDNumberLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
